app.js

```diff
@@ -0,0 +1,205 @@
+  },
+  {
+    title: 'Budget App',
+    info: ['facebook', 'full stack dev', '2015'],
+    description:
+      'The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.',
+    skills: ['ruby', 'rails', 'bootstrap'],
+    image: 'images/wishyacht.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Companies 3D',
+    info: ['frontend', 'Api', '2022'],
+    description:
+      "this is an informative/numeric web application that shows information about the canadian technical companies and their statistcs like the market cap and the trading status it also has search feature and for each company page it's unique root and responsive website",
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/company.png',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Space hub',
+    info: ['pair', 'APIs', '2022'],
+    description:
+      'A web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions',
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/space.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+];
+
+const btns = document.querySelectorAll('.work button');
+const bgModal = document.querySelector('.bg-modal');
+
+function popUp(index) {
+  const {
+    title, info, description, skills, image,
+  } = popupCards[index];
+
+  const infoHtml = info
+    .map(
+      (i) => `
+      <li class="canopy">${i}</li>
+      ${
+  i === info[info.length - 1]
+    ? ''
+    : '<li><img src="images/Counter.png" alt="counter" /></li>'
+}
+    `,
+    )
+    .join('');
+
+  const skillsHtml = skills
+    .map((s) => `<li class="highlight font-1">${s}</li>`)
+    .join('');
+
+  bgModal.innerHTML = `
+  <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="tonic">${title}</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class="ul-box1 flex0">
+          ${infoHtml}
+        </ul>
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="flex-99">
+          <div class="flex-98">
+            <p class="description-2 flex0">
+            ${description}
+            </p>
+          </div>
+          <div>
+            <ul class="ul-box2 flex0">
+             ${skillsHtml}
+            </ul>
+            <hr class="solid-1">
+            <div class="buttons">
+              <button type="button" class="live-btn">
+                see live
+                <i class="fas fa-check"></i>
+              </button>
+              <button type="button" class="live-btn">
+                see source
+                <i class="fab fa-github"></i>
+              </button>
```

Comment:
- [ ] it would be better if you use the same icon in the button as in the Figma design the see live is different please try to fix it 

- [ ]  also you should Add effect to the button when hovering and pressing as in the Figma design 
- Yours 
<img width="260" alt="Screen Shot 2022-10-20 at 17 09 28" src="https://user-images.githubusercontent.com/61790587/197001695-db5548bc-b54e-4adb-8bb8-3ab9e3843543.png">

- Figma
<img width="177" alt="Screen Shot 2022-10-19 at 17 58 59" src="https://user-images.githubusercontent.com/61790587/197001500-a0e6198a-d054-4c2f-a214-f9f92754b399.png">
<img width="577" alt="Screen Shot 2022-10-19 at 18 03 04" src="https://user-images.githubusercontent.com/61790587/197001518-46dd0aa5-37ef-4ec4-937f-204c1e611466.png">